src/reducers.js

```diff
@@ -60,7 +61,7 @@ function organisations(
     timesFetched: 0,
     available: [],
     availableForRasterSharedWith: [],
-    selected: getLocalStorage("lizard-management-current-organisation", null),
+    selected: null,
```

Comment:
I could remove "getLocalStorage("lizard-management-current-organisation" here because I dispatch it in the action instead.
Dispatching it in the action allows me to also trigger a fetch to the usage api on startup.